openQA.spec

```
@@ -238,6 +239,16 @@ Group:          Development/Tools/Other
 Documentation material covering installation, configuration, basic test writing, etc.
 Covering both openQA and also os-autoinst test engine.
 
+%package auto-upgrade
+Summary:        Automatically upgrade and reboot an openQA system where required
```

An "openQA system"? Instead of introducing a new term I'd just use "the system".

```suggestion
Summary:        Automatically upgrade and reboot the system when required
```

---

This line hasn't changed yet.

---

oh, good catch. Have confused the summary with description